src/components/learner-credit-management/tests/BudgetDetailPage.test.jsx

```diff
@@ -426,7 +443,20 @@ describe('<BudgetDetailPage />', () => {
       budgetRedemptions: mockEmptyBudgetRedemptions,
       fetchBudgetRedemptions: jest.fn(),
     });
-
+    useEnterpriseCustomer.mockReturnValue({
```

Comment:
Do we need to include this exact mocking for the hooks before every single test?

Reply:
yes, the tests fail without it.

Reply:
Can we at least move the boilerplate setup into a single-line method?  As-is, if the mocking setup changes in the future it's going to be a royal pain to update.